lib/cli/src/commands/package/tag.rs

```diff
@@ -66,11 +68,14 @@ pub struct PackageTag {
     #[clap(name = "hash")]
     pub package_hash: PackageHash,
 
-    ///
+    /// The package to tag.
+    #[clap(name = "package_ident")]
+    pub package_id: Option<NamedPackageIdent>,
+
     /// Directory containing the `wasmer.toml`, or a custom *.toml manifest file.
     ///
     /// Defaults to current working directory.
-    #[clap(name = "path", default_value = ".")]
+    #[clap(long = "manifest-path", default_value = ".")]
```

Comment:
I am not sure calling it manifest-path is the right approach. It makes things confusing. 

```
      --dir <DIR>
          The path to the directory containing the `app.yaml` file
```

wasmer publish has

```
  [path]
          Directory containing the `wasmer.toml`, or a custom *.toml manifest file.
```

Reply:
Regarding the `--dir` flag in `deploy`: https://github.com/wasmerio/wasmer/issues/4845. I'll rename `--manifest-path` to `--path` in `tag - I think the former is more descriptive but longer (and breaking), so let's just keep the symmetry.

Reply:
sgtm